bokehjs/src/lib/api/themes.ts

```diff
@@ -209,7 +209,7 @@ export const caliber = new Theme([
   }),
 ])
 
-export const constrast = new Theme([
+export const contrast = new Theme([
```

Comment:
It might be good to make two separate PRs for the changes that affect code and the ones that affect only comments,  docs, and error messages.  That way any potential breakages can be localized to a much smaller set of changes.

Reply:
I added a deprecation for the previous name.

Reply:
Given this is the only API change, I will keep it in this PR.